Simplified/NYPLAlertUtils.swift

```diff
@@ -38,13 +39,17 @@ import UIKit
       }
     }
     #endif
-    
-    if customMessage != nil { // Custom message override
-      message = customMessage as! String
-    } else if message.isEmpty { // Handle unassigned message
-      message = "UnknownError"
+
+    if message.isEmpty {
+      // since it wasn't a networking or Adobe DRM error, show the error
+      // description if present
+      if let errorDescription = error?.localizedDescription, !errorDescription.isEmpty {
+        message = errorDescription
+      } else {
+        message = "UnknownError"
```

Comment:
We should consider logging cases when this happens since it is unexpected. This way we can be warned via Crashlytics when we have missing localizations for errors. Can we fall back to english errors if localization is missing or does `localizedDescription` do that already?

Reply:
If the NSLocalizedDescriptionKey is populated in `userInfo`, it will be used no matter what. And if it's constructed with NSLocalizedString("key"...) (as it should) then if "key" is not localized it will default to "key" as the not-localized value.

I'll see if I can add a logging call there since it would be nice to avoid that UnknownError.